app/src/main/java/com/github/houseorganizer/houseorganizer/panels/offline/OfflineScreenActivity.java

```diff
@@ -62,4 +110,17 @@ public void unsupportedActionAlert(View view) {
                 .setMessage("This action is not available at the moment")
                 .show();
     }
+
+    public void goOnlineIfPossible(View view) {
+        ConnectivityManager connectivityManager = (ConnectivityManager) getSystemService(Context.CONNECTIVITY_SERVICE);
+        NetworkInfo activeNetInfo = connectivityManager.getActiveNetworkInfo();
+
+        boolean isConnected = (activeNetInfo != null) && activeNetInfo.isConnectedOrConnecting();
+
+        if (isConnected) {
+            startActivity(new Intent(this, MainScreenActivity.class));
+        } else {
+            unsupportedActionAlert(view);
```

Comment:
The message would be clearer if changed to "You need to have an active connexion to go back to online mode." or something like that.

Reply:
I agree and will make the changes shortly